umap/static/umap/js/modules/help.js

```diff
@@ -190,42 +190,48 @@ export default class Help {
   show(entries) {
     const container = DomUtil.add('div')
     DomUtil.createTitle(container, translate('Help'))
-    // Special dynamic case. Do we still think this dialog is usefull ?
-    if (entries === 'edit') {
+    for (const name of entries) {
       DomUtil.element({
         tagName: 'div',
         className: 'umap-help-entry',
         parent: container,
-      }).appendChild(this._buildEditEntry())
-    } else {
-      for (const name of entries) {
-        DomUtil.element({
-          tagName: 'div',
-          className: 'umap-help-entry',
-          parent: container,
-          innerHTML: ENTRIES[name],
-        })
-      }
+        innerHTML: ENTRIES[name],
+      })
     }
     this.map.dialog.open({ content: container, className: 'dark' })
   }
 
+  // Special dynamic case. Do we still think this dialog is usefull ?
+  showGetStarted() {
+    const container = DomUtil.add('div')
+    DomUtil.createTitle(container, translate('Where do we go from here?'))
+    DomUtil.element({
+      tagName: 'div',
+      className: 'umap-help-entry',
+      parent: container,
+    }).appendChild(this._buildEditEntry())
+    this.map.dialog.open({ content: container, className: 'dark' })
+  }
+
   button(container, entries, classname) {
     const button = DomUtil.createButton(
       classname || 'umap-help-button',
       container,
       translate('Help')
     )
     entries = typeof entries === 'string' ? [entries] : entries
-    DomEvent.on(button, 'click', DomEvent.stop).on(button, 'click', () =>
+    DomEvent.on(button, 'click', DomEvent.stop).on(button, 'click', () => {
       this.show(entries)
-    )
+    })
     return button
   }
 
-  link(container, entries) {
-    const button = this.button(container, entries, 'umap-help-link')
+  getStartedLink(container) {
+    const button = DomUtil.createButton('umap-help-link', container, translate('Help'))
     button.textContent = translate('Help')
+    DomEvent.on(button, 'click', DomEvent.stop).on(button, 'click', () => {
```

Comment:
What are we stopping prior to show the element? Would it be easier to use regular JS event handling?